### PR TITLE
Update bubble scan portfolio

### PR DESCRIPTION
**Pull request by Chirag2x**
Updates on Tech lead and Development Priorities.

### CODE REVIEW

File: docs/project_bubblescan/about.md [hunk 1]
```diff
@@ -16,8 +16,9 @@ The chemistry department at SLU uses paper scantron (fill-in-the-bubble) sheets
 - **Source Code:** [https://github.com/oss-slu/bubble_scan](https://github.com/oss-slu/bubble_scan) [<img src="/img/git-alt.svg" alt="git" width="25" height="25" />](https://github.com/oss-slu/bubble_scan)
 
 - **Client** Dr. Brian Woods
-- **Current Tech Lead:** Raju Karmuri [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/rkarmuri) [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/rajukarmuri731/)
+- **Current Tech Lead:** Chirag Gupta [<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/Chirag2x) [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/chirag-gupta-3911961b4/)
 - **Developers:**
+  - Raju Karmuri (Alumni, Previous Tech lead)[<img src="/img/github.svg" alt="github" width="25" height="25" />](https://github.com/rkarmuri) [<img src="/img/linkedin.svg" alt="linkedin" width="25" height="25" />](https://www.linkedin.com/in/rajukarmuri731/)
```
Comment (kungfuchicken) [inline docs/project_bubblescan/about.md:21]: please use all lower case for the tags, and use "prior" instead of "previous": `(alumni, prior tech lead)`.

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
almost there.

File: docs/project_bubblescan/about.md (hunk 1 above)
Comment (kungfuchicken) [reply docs/project_bubblescan/about.md:21]: hey, any updates?

Comment (Chirag2x) [reply docs/project_bubblescan/about.md]: Hii Daniel, I made the recommended changes.

**Review by kungfuchicken — COMMENTED** [review]
cool!

**Review by kungfuchicken — APPROVED** [review]
lgtm;

**Pull request merged**